cleanlab/classification.py

```diff
@@ -323,14 +324,14 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
           that are mislabeled examples from every other class ``k_y``,
           Assumes columns of `inverse_noise_matrix` sum to 1.
 
-        label_issues : pd.DataFrame or np.array<bool> or np.array<int>, optional
+        label_issues : pd.DataFrame or np.array, optional
```

Comment:
The doc wasn't rendering the type anyways. Python syntax for type parameters uses square brackets. I dropped the parameters here rather than fixing them to be consistent with the rest of the docs.

I think we should consider adding type parameters in the future, with static type checking with mypy.